### PR TITLE
added multiavatar api to existing random avatar generators

### LINKED ISSUE #46: Add more sources of random images
Currently following APIs are used

* Dicebear
* Robohash

Would be nice to have more sources of random images. The following needs to be updated

* Add a new function to generate a proper link to the new image source API on `utils.rs`
* `generate_random_avatar_link` should be updated with an option to call the new function.

### PR DESCRIPTION
**Pull request by aybeedee**
# Issue

Closes #46 

# Description

Added [Multiavatar](https://api.multiavatar.com/) API to the existing random avatar generator options.

### CODE REVIEW

File: gui/src/utils.rs [hunk 1]
```diff
@@ -62,6 +67,7 @@ pub fn generate_random_avatar_link() -> String {
     match selected_choice {
```
Comment (TheRustyPickle) [inline gui/src/utils.rs:67]: The `choices` array needs to be updated with `multiavatar` otherwise this choice would never be selected.

**Review by TheRustyPickle — CHANGES_REQUESTED** [review]

File: gui/src/utils.rs (hunk 1 above)
Comment (aybeedee) [reply gui/src/utils.rs:67]: Overlooked it! Made the change now.

**Review by TheRustyPickle — APPROVED** [review]

**Comment by TheRustyPickle** [comment]
Thank you for your contribution, @aybeedee! You can take a look at the other issues if you are interested in contributing further and drop a star if you like the project!

**Comment by aybeedee** [comment]
@TheRustyPickle I'll definitely go through them if I get more time. Also, I'm very new to Rust. What's up with the emphasis on snake case? Even the compiler suggests me to use it! I'm very used to camel and pascal case myself. Is there a particular reason for the preference? Or am I just too much of a newbie to this ecosystem.

**Comment by TheRustyPickle** [comment]
@aybeedee I believe it is the official Rust style of writing functions, variables, etc. The use of the snake case is emphasized in the [official Rust book](https://doc.rust-lang.org/book/ch03-03-how-functions-work.html#functions) as well.

**Pull request merged**